devel/ci/integration/bodhi/Dockerfile-f32

```diff
@@ -6,7 +6,7 @@ LABEL \
   license="MIT"
 
 # For integration testing we're using the infrastructure repo
-RUN curl -o /etc/yum.repos.d/infra-tags.repo https://infrastructure.fedoraproject.org/cgit/ansible.git/plain/files/common/fedora-infra-tags.repo
+RUN curl -o /etc/yum.repos.d/infra-tags.repo https://pagure.io/fedora-infra/ansible/raw/master/f/files/common/fedora-infra-tags.repo
 # work around https://bugzilla.redhat.com/show_bug.cgi?id=1699396
```

Comment:
We can remove the workaround from all the dockerfiles, since they're all F>31

Reply:
Ha I missed this one :) updating the PR now